client/apps/edit/components/admin/verticals_tags/index.styl

```diff
@@ -0,0 +1,15 @@
+.edit-admin--verticals-tags
+  button
+    margin 5px 8px 8px 0px
+    border-radius 1px
+    padding 13px 15px
+    &.disabled
+    	background-color gray-lightest-color
+    	&:hover
+    		color black
+
+  .fields-left
+    .avant-garde-button-black
+      background black !important
```

Comment:
Generally try to avoid !important. I bet you could select it using something more specific than .avant-garde-button-black